mqtt-v5-broker/src/broker.rs

```diff
@@ -178,7 +209,9 @@ pub enum WillDisconnectLogic {
 
 #[derive(Debug)]
 pub enum BrokerMessage {
+    Stats,
```

Comment:
This feels a bit unrelated to the topic of introducing plugins to the broker, could you separate it from this PR?

Reply:
Sure. I removed it.

For metrics I would sugest  [prometheus](https://prometheus.io) - but that's as you noted unrelated to this discussion.